app/support/template.hbs

```diff
@@ -1,5 +1,5 @@
 {{title (t 'support.title')}}
-<div class="container">
+<div class="Support container">
```

Comment:
Not much benefit to adding a BEM class to the root element, unless you're putting styles on it directly. The stylesheet is already scoped to the route.

Reply:
It only seems to work for components. Routes appear to be global.

Reply:
In `dist/assets/ember-osf-web.css` I see
```css
.__support__bd682 .Support a:not(.btn),
.__support__bd682 .Support .btn-link {
  font-weight: bold; }
```